Makefile

```diff
@@ -120,13 +120,18 @@ debug:
 
 ifneq ($(OS),Windows_NT)
 extra_run_flags = -v /:/hostfs:ro -v /var/run/docker.sock:/var/run/docker.sock:ro -v /tmp/scratch:/tmp/scratch
+ifeq ($(PLATFORM),Linux)
+extra_run_flags += -v /var/run/dbus/system_bus_socket:/var/run/dbus/system_bus_socket:ro
```

Comment:
Remove all this.  You can have an envvar that is pasted directly into the `docker exec` below that you can specify to add these additional flags to the dev image but let's not make this any more complicated than it already is for the sake of a single plugin.